docs/getting-started/index.rst

```diff
@@ -11,85 +11,163 @@ Getting started
+    ^^^^^^^^^^^^^^^^^^^
+
+    Gammapy can be installed with `Anaconda <https://docs.continuum.io/anaconda/>`__ or Miniconda:
+
+    .. code-block:: bash
+
+        conda install gammapy
 
     ---
-    :img-top: ../_static/install.png
 
-    Installation
-    ^^^^^^^^^^^^
+    Prefer pip?
+    ^^^^^^^^^^^
+
+    Gammapy can be installed via pip from `PyPI <https://pypi.org/project/gammapy/>`__.
 
-    Install and set-up.
 
-    +++
+    .. code-block:: bash
 
-    .. link-button:: installation
-            :type: ref
-            :text: To the installation
-            :classes: btn-block btn-secondary stretched-link
+        pip install gammapy
 
     ---
-    :img-top: ../_static/using.png
+    :column: col-12 p-3
 
-    Using Gammapy
-    ^^^^^^^^^^^^^
+    In-depth instructions?
+    ^^^^^^^^^^^^^^^^^^^^^^
 
-    Different ways on how to use Gammapy.
+    Working with virtual environments? Installing a specific version? Installing from source?  Check the advanced
+    installation page.
 
-    +++
+    .. link-button:: install
+        :type: ref
+        :text: Learn more
+        :classes: btn-secondary stretched-link
 
-    .. link-button:: usage
-            :type: ref
-            :text: To the usage items
-            :classes: btn-block btn-secondary stretched-link
 
-    ---
-    :img-top: ../_static/gears.png
+Introduction
+------------
 
-    Troubleshooting
-    ^^^^^^^^^^^^^^^
+.. accordion-header::
+    :id: collapseTwo
+    :title: How to access gamma-ray data
+    :link: ../tutorials/data/hess.html#DL3-DR1
 
-    In case you may have install issues.
+To access IACT data in the DL3 format, use the `~gammapy.data.DataStore`. It allows
+easy access to observations stored in the DL3 data library.
+It is also internally used by the high level interface `~gammapy.analysis.Analysis`.
 
-    +++
+.. accordion-footer::
 
-    .. link-button:: troubleshoot
-            :type: ref
-            :text: To the troubleshooting
-            :classes: btn-block btn-secondary stretched-link
+.. accordion-header::
+    :id: collapseSeven
+    :title: How to compute a 1D spectrum
+    :link: ../tutorials/analysis/1D/spectral_analysis.html
 
-    ---
-    :img-top: ../_static/upgrade.png
+The `~gammapy.analysis.Analysis` class can perform spectral extraction. The
+`~gammapy.analysis.AnalysisConfig` must be defined to produce '1d' datasets.
+Alternatively, you can follow the spectral analysis tutorial.
 
-    Upgrade
-    ^^^^^^^
+.. accordion-footer::
 
-    A short recipe on how to upgrade.
+.. accordion-header::
+    :id: collapseSix
+    :title: How to compute a 2D image
+    :link: ../tutorials/index.html#d-image
 
-    +++
+Gammapy treats 2D maps as 3D cubes with one bin in energy. Sometimes, you might want to use previously
+obtained images lacking an energy axis (eg: reduced using traditional IACT tools) for modeling and fitting
+inside Gammapy. In this case, it is necessary to attach an `energy` axis on as it is showm in the tutorials.
```

Comment:
shown*

Reply:
Do we have such an example in the tutorials? I started to introduce this here https://github.com/gammapy/gammapy/pull/3350, but then we decided to have it in the internal HESS documentation.

Reply:
I think the perfect solution would be something like Pandas has done here: https://pandas.pydata.org/docs/getting_started/index.html#coming-from

So sub-pages like `Coming from HESS`, `Coming from CTA`, `Coming from MAGIC`, etc. But this could be a separate PR...